content/en/docs/ambient/upgrade/helm/revision_test.sh

```diff
@@ -49,5 +49,5 @@ _rewrite_helm_repo snip_rollback_tag
 # upgrading a tag creates an MWC, let's clean it up 
 export REVISION=istio-1-22-1
 helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags="{tagname}" --set revision="$OLD_REVISION" -n istio-system | kubectl delete -f -
-helm delete istiod-"$REVISION" -n istio-system
+helm delete istiod-"$OLD_REVISION" -n istio-system
```

Comment:
I think we need to delete everything here in `cleanup` - $REVISION as well as $OLD_REVISION.

I think this won't remove $REVISION istiod anymore?

Reply:
TBH, I'm not familiar with the doctests. Let me add cleanup for $REVISION as well.

Reply:
Thanks for reviewing @bleggett. All tests are passing, hope its all good :D